api/src/startup.rs

```diff
@@ -92,6 +92,21 @@ impl Application {
         Ok(())
     }
 
+    pub async fn delete_all_test_databases(config: Settings) -> Result<(), anyhow::Error> {
+        let mut connection = PgConnection::connect_with(&config.database.without_db()).await?;
+        let databases = connection
+        .fetch_all(&*r#"select datname from pg_database where datname not in ('postgres', 'template0', 'template1');"#.to_string())
```

Comment:
```suggestion
        .fetch_all(&*r#"select datname from pg_catalog.pg_database where datname not in ('postgres', 'template0', 'template1');"#.to_string())
```

It is better to use fully-qualified names to avoid [CVE-2018-1058](https://wiki.postgresql.org/wiki/A_Guide_to_CVE-2018-1058%3A_Protect_Your_Search_Path), although ideally it is good to lock `search_path` as suggested by the wiki page.

Reply:
Done, thanks.